diff --git a/internal/engine/engine.go b/internal/engine/engine.go
--- a/internal/engine/engine.go
+++ b/internal/engine/engine.go
@@ -188,5 +188 @@
-	var status mutator.Status
-
-	if mu.codeData.Cov.IsCovered(pos) {
-		status = mutator.Runnable
-	}
+	status := mutator.Runnable
diff --git a/internal/engine/executor.go b/internal/engine/executor.go
--- a/internal/engine/executor.go
+++ b/internal/engine/executor.go
@@ -228 +228 @@ func (m *mutantExecutor) getTestArgs(pkg string) []string {
-		args = append(args, fmt.Sprintf("-cpu %d", m.testCPU))
+		args = append(args, "-cpu", fmt.Sprint(m.testCPU))
